Content.Client/Audio/BackgroundAudioSystem.cs

```diff
@@ -134,23 +134,26 @@ private void CheckAmbience(TransformComponent xform)
         private void EntParentChanged(ref EntParentChangedMessage message)
         {
             if(_playMan.LocalPlayer is null || _playMan.LocalPlayer.ControlledEntity != message.Entity ||
-               !_timing.IsFirstTimePredicted) return;
+               !_timing.IsFirstTimePredicted)
+                return;
 
             // Check if we traversed to grid.
             CheckAmbience(message.Transform);
         }
 
         private void ChangeAmbience(SoundCollectionPrototype newAmbience)
         {
-            if (_currentCollection == newAmbience) return;
+            if (_currentCollection == newAmbience)
+                return;
             _timerCancelTokenSource.Cancel();
             _currentCollection = newAmbience;
             _timerCancelTokenSource = new();
             Timer.Spawn(1500, () =>
             {
                 // If we traverse a few times then don't interrupt an existing song.
-                if (_playingCollection == _currentCollection) return;
-                EndAmbience();
+                // If we are in the Lobby, don't start Ambience because of a player movement
+                if (_playingCollection == _currentCollection || _stateManager.CurrentState is LobbyState)
```

Comment:
Shouldn't this check if it isn't game state?

Reply:
Yes. fixed